lua/terrortown/entities/roles/sidekick/shared.lua

```diff
@@ -6,9 +6,10 @@ if SERVER then
 	resource.AddFile("materials/vgui/ttt/dynamic/roles/icon_siki.vmt")
 
 	CreateConVar("ttt2_siki_protection_time", 1, {FCVAR_NOTIFY, FCVAR_ARCHIVE})
-	CreateConVar("ttt2_siki_mode", 1, {FCVAR_NOTIFY, FCVAR_ARCHIVE})
 end
 
+CreateConVar("ttt2_siki_mode", 1, {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED}, "0 = Sidekick doesn't become Jackal and can't win alone, but gets targets / 1 = Sidekick becomes Jackal on Jackals death / 2 = Sidekick doesn't become Jackal but can win alone and gets targets", 0, 2)

```

Comment:
The Sidekick works with roles which are not the Jackal as well.
I guess the descriptions should be updated to a more general explanation.

Reply:
There is a description of all possibilities on the workshop page:
![image](https://user-images.githubusercontent.com/13639408/90106577-ec97a880-dd47-11ea-8035-bdaa36b79713.png)

Reply:
Isn't it currently that it is impossible for the ttt2_siki_mode convar to set the mode specifically to the jester/clairvoyant or Jackal?
But yes, i agree with the generalization. I'll change that.

Reply:
done with dd4f4af981af7942c2224aa5787f25dba9b6e4bc